cmd/influxd/run/query_executor.go

```diff
@@ -0,0 +1,57 @@
+package run
+
+import (
+	"github.com/influxdata/influxdb/cluster"
+	"github.com/influxdata/influxdb/influxql"
+	"github.com/influxdata/influxdb/monitor"
+	"github.com/influxdata/influxdb/services/meta"
+	"github.com/influxdata/influxdb/tsdb"
+)
+
+// NewQueryExecutor returns an instance of QueryExecutor with registered statement executors.
+func NewQueryExecutor(mc *meta.Client, s *tsdb.Store, m *monitor.Monitor, pw *cluster.PointsWriter) *influxql.QueryExecutor {
```

Comment:
`PointsWriter` not needed? Was confused why this was getting passed into a query executor